src/hs_config.h

```diff
@@ -72,10 +73,11 @@ typedef struct hs_config
 
   unsigned max_message_size;
   unsigned output_size;
-  unsigned analysis_threads;
   unsigned backpressure;
   unsigned backpressure_df;
   int      pid;
+  uint8_t  analysis_threads;
```

Comment:
Would we want to also update the parameter type for `hs_cleanup_checkpoints`

https://github.com/mozilla-services/hindsight/blob/19f24ae33c519ee11b3417d4be52e550ab6d5eed/src/hs_checkpoint_reader.c#L308
https://github.com/mozilla-services/hindsight/blob/19f24ae33c519ee11b3417d4be52e550ab6d5eed/src/hs_checkpoint_reader.h#L84

Reply:
Yeah I will make it consistent, thanks.